chef_master/source/knife_setup.rst

```diff
@@ -58,6 +58,77 @@ Generating a config.rb File
 
 The knife command `knife configure` can be used to generate your initial config.rb configuration file in your home directory. See `knife configure </knife_configure.html>`_ for details
 
+Knife Profiles
+=====================================================
+
+Profile support since Chef 13.7
```

Comment:
I don't know if there's a tag or pattern for this?

Reply:
`**Profile Support since Chef 13.7**`